src/components/Nav.jsx

```diff
@@ -18,7 +18,8 @@ function Nav() {
     return (
         <nav>
             <ul id="topics_ul">
-                {topics.map(topic => <li key={topic.slug}><Link to={topic.slug} id='topic' className={'topic' + '_' + topic.slug}>{topic.slug}</Link></li>)}
+                <li key='Home'><Link to='/' id='topic' className={'topic' + '_' + 'home'}>All</Link></li>

```

Comment:
try using a template literal here and on the following line to get rid of the react error message